draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1749,13 +1755,18 @@ Practical user agent implementations have limits on the number and size of
 cookies that they can store. General-use user agents SHOULD provide each of the
 following minimum capabilities:
 
-*   At least 4096 bytes per cookie (as measured by the sum of the length of the
-    cookie's name, value, and attributes).
-
 *   At least 50 cookies per domain.
 
 *   At least 3000 cookies total.
 
+User agents MAY limit the maximum number of cookies they store, and may evict
+any cookie at any time (whether at the request of the user or due to
```

Comment:
nit: can you delete the "on orders from the user" on line 1776 since it now mentions that here?

Reply:
done.